pairinteraction/CMakeLists.txt

```diff
@@ -140,6 +140,26 @@ if (NOT MKL_FOUND AND PKG_CONFIG_FOUND)
   endif()
 endif()
 
+function(target_link_mkl TARGET)
+  if(MSVC AND MKL_FOUND AND MKL_ROOT)
+    set(MKL_REDIST_DIR ${MKL_ROOT}/redist/intel64)
+    set(MKL_DLLS
+      ${MKL_REDIST_DIR}/mkl_avx.2.dll
+      ${MKL_REDIST_DIR}/mkl_avx2.2.dll
+      ${MKL_REDIST_DIR}/mkl_avx512.2.dll
+      ${MKL_REDIST_DIR}/mkl_core.2.dll
+      ${MKL_REDIST_DIR}/mkl_def.2.dll
+      ${MKL_REDIST_DIR}/mkl_sequential.2.dll)
```

Comment:
Not sure whether that is really the set of necessary and sufficient libraries, but it works.

Reply:
Maybe also this?
```suggestion
    set(MKL_DLLS
      ${MKL_REDIST_DIR}/mkl_avx.2.dll
      ${MKL_REDIST_DIR}/mkl_avx2.2.dll
      ${MKL_REDIST_DIR}/mkl_avx512.2.dll
      ${MKL_REDIST_DIR}/mkl_core.2.dll
      ${MKL_REDIST_DIR}/mkl_def.2.dll
      ${MKL_REDIST_DIR}/mkl_${MKL_THREADING}.2.dll)
```